datafusion/physical-plan/src/streaming.rs

```diff
@@ -58,7 +60,9 @@ pub struct StreamingTableExec {
     projected_schema: SchemaRef,
     projected_output_ordering: Vec<LexOrdering>,
     infinite: bool,
+    limit: Option<usize>,
```

Comment:
Can we please also add `limit` to the display / explain plan? 

https://github.com/alamb/datafusion/blob/b8d7498da504956ee62fb0b3722f7c46e0831466/datafusion/physical-plan/src/streaming.rs#L171-L201

Reply:
Done